libc/include/math.h.def

```diff
@@ -11,6 +11,26 @@
 
 #include <__llvm-libc-common.h>
 #include <llvm-libc-macros/math-macros.h>
+#include <llvm-libc-types/float128.h>
+
+#ifdef LIBC_COMPILER_HAS_FLOAT128
```

Comment:
is the macro exposed to user interface?

Reply:
Yes they are.  All the headers inside `libc/include` are expected to be used by users in full build mode.